docs/agents/browser.md

```diff
@@ -0,0 +1,123 @@
+
+There are two parts of the RUM script: 
+
+ - The loader script,
+ - The metrics gathering script itself.
+
+The loader script is responsible for loading the main, metrics gathering script itself and adds the actual `<script>` tags with the content on the main metric script. 
+
+The loader script will keep track and buffer any of the commands that need to be sent after the main metrics gathering script will be loaded and configured. When the main RUM script finally loads it will read any pending
+commands will be executed.
+
+### Working on the Loader Script
+
+The main entry point of the loader script is located in `src/snippet.js`. This is the unminified [ES2015]((https://en.wikipedia.org/wiki/ECMAScript)) version of the loader script which is responsible for loading the main RUM script. 
+
+To generate the minified snippet of the loader script use `yarn run generate-snippet`.
+
+### Working on the Main RUM Script
+
+The main entry point of the RUM script is located in `src/index.js`. Use `eslint` to catch any style issues and
+`flow` to catch any type errors.
+
+You can use the following commands:
+
+ - `yarn run start` to start the development server
+ - `yarn run lint` to run `lint` on the code
+ - `yarn run flow` to run `flow` type checks
+ - `yarn run build` to build the production bundle
+ - `yarn run dev` to build the development (not minified) bundle
+
+### Supported Commands
+
+The following commands are currently supported by the RUM script:
+
+ - `config` - Command used to configure the RUM script. It requires the `token` and `receiverUrl` configuration parameters. You can also provide the `ignoreBotData` Boolean parameter to ignore metrics coming from bots.   
+ - `identify` - Command used to identify the user. Requires the `name` and the `identifier` keys to be present, which are used to identify a user.  
+ - `routeChange` - Command used to inform the RUM script that a route change happened. It accepts a single parameter that should take the value of the new route. 
+ - `startTransaction` - Command used to inform the RUM script that a transaction starts. It accepts a single parameter that should take the value of the transaction name.
+ - `endTransaction` - Command used to inform the RUM script that a transaction ends. It accepts a single parameter that should take the value of the name of the transaction that we want to end.
+ - `pageLoad` - Command used to provide metrics related to page load.
+ - `ajax` - Command used to provide metrics related to HTTP requests.
+ - `longTask` - Command used to provide metrics about [long tasks](/experience/longtasks/).
+ - `elementTiming` - Command used to provide metrics about [element timing](/experience/element-timing/).
+ - `vital-metric` - Command used to provide metrics about [web vitals](/experience/webvitals/).
+
+### Global Variables
+
+The RUM script uses global constants defined in the `constants.js` file. The file contains the following constants:
```

Comment:
Constants or variables?  Let's use the same term.

Reply:
Constants. Adjusted.